src/main/java/com/ubc/cpsc304/hotelify/controller/dto/ReviewResponseDto.java

```diff
@@ -0,0 +1,8 @@
+package com.ubc.cpsc304.hotelify.controller.dto;
+
+import lombok.Data;
+
+@Data
+public class ReviewResponseDto {
+    private Long review_id;
```

Comment:
we probably want to return more than that, say if we want to get all reviews from db, only ids won't make sense, which means we need stuff like `comment` and `rating`